.github/workflows/clt_nightly.yml

```diff
@@ -22,6 +22,7 @@ on:
       - 'test/clt-tests/integrations/**'
       - 'test/clt-tests/core/test-performance-for-queries-with-multiple-disk-chunks'
       - 'test/clt-tests/integrations/**'
+      - 'test/clt-tests/native-support-for-message-queues/**'
```

Comment:
update path to ....integrations/kafka/**

Reply:
Done